From e271879b3b455dad5b2914f6ce7e1152864f5ba9 Mon Sep 17 00:00:00 2001
From: Nikita Poltorapavlo <Nikita_Poltorapavlo2@cable.comcast.com>
Date: Mon, 29 Nov 2021 18:56:31 +0200
Source: COMCAST
License: Apache-2.0
Upstream-Status: Pending
Subject: [PATCH] RDKTV-11792: increase retry count

Reason for change: Internet precondition is never met
if the network connection is not up in first 200s.
Test Procedure: Start with network off, connect
after 5-6 min.
Risks: Low
Signed-off-by: Nikita Poltorapavlo <npoltorapavlo@productengine.com>
---
 LocationSync/LocationSync.config       | 2 +-
 LocationSync/LocationSyncPlugin.json   | 2 +-
 LocationSync/doc/LocationSyncPlugin.md | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/LocationSync/LocationSync.config b/LocationSync/LocationSync.config
index ea7584a..1478bcc 100644
--- a/LocationSync/LocationSync.config
+++ b/LocationSync/LocationSync.config
@@ -4,7 +4,7 @@ set (preconditions Network)
 
 map()
     kv(interval 10)
-    kv(retries 20)
+    kv(retries 60)
     kv(source ${PLUGIN_LOCATIONSYNC_URI})
 end()
 ans(configuration)
diff --git a/LocationSync/LocationSyncPlugin.json b/LocationSync/LocationSyncPlugin.json
index d016e8f..fca6178 100644
--- a/LocationSync/LocationSyncPlugin.json
+++ b/LocationSync/LocationSyncPlugin.json
@@ -23,7 +23,7 @@
                     "retries": {
                         "type": "number",
                         "size": 8,
-                        "description": "Maximum number of request reties to the Location Server (default:20)."
+                        "description": "Maximum number of request reties to the Location Server (default:60)."
                     },
                     "source": {
                         "type": "string",
diff --git a/LocationSync/doc/LocationSyncPlugin.md b/LocationSync/doc/LocationSyncPlugin.md
index db82249..81491fd 100644
--- a/LocationSync/doc/LocationSyncPlugin.md
+++ b/LocationSync/doc/LocationSyncPlugin.md
@@ -79,7 +79,7 @@ The table below lists configuration options of the plugin.
 | autostart | boolean | Determines if the plugin shall be started automatically along with the framework |
 | configuration | object | <sup>*(optional)*</sup>  |
 | configuration?.interval | number | <sup>*(optional)*</sup> Maximum time duration between each request to the Location Server (default: 10) |
-| configuration?.retries | number | <sup>*(optional)*</sup> Maximum number of request reties to the Location Server (default:20) |
+| configuration?.retries | number | <sup>*(optional)*</sup> Maximum number of request reties to the Location Server (default:60) |
 | configuration?.source | string | <sup>*(optional)*</sup> URI of the Location Server (default:"location.webplatformforembedded.org") |
 
 <a name="head.Interfaces"></a>
-- 
2.25.1

